src/api/entities/BusinessContact.ts

```diff
@@ -12,12 +12,16 @@ export class BusinessContact extends BaseModel {
   static get tableName() {
     return 'businessContacts';
   }
-  static relationMappings = {
+  static readonly relationMappings = {
     participant: {
       relation: Model.BelongsToOneRelation,
       modelClass: 'Participant',
       join: {
-        from: 'users.participantId',
+        from: 'users.id',
```

Comment:
I don't think users.id was ever right here. A business contact belongs to a participant, not a user.

Reply:
Fixed this to just join the Participants table directly